src/strtab.rs

```diff
@@ -93,7 +93,7 @@ impl<'a> Index<usize> for Strtab<'a> {
     /// **NB**: this will panic if the underlying bytes are not valid utf8, or the offset is invalid
     #[inline(always)]
     fn index(&self, offset: usize) -> &Self::Output {
-        get_str(offset, &self.bytes, self.delim).unwrap()
+        self.get(offset).unwrap().unwrap()
```

Comment:
Ah yea that's why I did that haha; I can't remember if the makefile is present but if so you can take make api and it will compile with different variations